src/config.rs

```diff
@@ -56,6 +56,12 @@ pub struct Config {
     /// JSON key: `openaiAPIKey`
     #[serde(rename = "openaiAPIKey")]
     pub openai_api_key: String,
+
+    /// The API Endpoint to be used.
+    /// JSON key: `openaiAPIEndpoint`
+    #[serde(rename = "openaiAPIEndpoint")]
```

Comment:
It's better to add a default value here, since most people probably won't change the endpoint.